docs/content/reports/ransomware.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Ransomware report"
+date: 2021-08-05T13:23:10+03:00
+draft: false
+description: "Provides information about ransomware simulation on your network"
+---
+
+{{% notice info %}}
+Check out [the documentation for other reports available in the Infection Monkey]({{< ref "/reports" >}}).
+{{% /notice %}}
+
+The Infection Monkey's **Ransomware Report** provides you with information and insight into ransomware simulation on your network.
```

Comment:
How about something like this?

The Infection Monkey can be configured to [simulate a ransomware attack](../usage/scenarios/ransomware-simulation/) on your network, whose findings can be found in the **Ransomware Report**.